x-pack/plugins/security_solution/public/management/components/endpoint_action_failure_message/endpoint_action_failure_message.tsx

```diff
@@ -35,24 +34,20 @@ export const EndpointActionFailureMessage = memo<EndpointActionFailureMessagePro
           if (
             endpointAgentOutput &&
             endpointAgentOutput.type === 'json' &&
-            endpointAgentOutput.content.code &&
-            endpointActionResponseCodes[endpointAgentOutput.content.code]
+            !!endpointAgentOutput.content.code &&
+            !!endpointActionResponseCodes[endpointAgentOutput.content.code]
           ) {
             errors.push(endpointActionResponseCodes[endpointAgentOutput.content.code]);
           }
         }
       }
 
+      if (action.errors && action.errors.length) {
+        errors.push(...new Set(action.errors));
```

Comment:
I see you removed the default `Action failed` message (fallback)... was that on purpose?
I feel like there should always be a value presented to the user... and if the code made it this far (got past 22-24), then even if for some reason (edge case) the `errors` is empty, we should still show a message since the overall action is Failed.

Reply:
This actually was redundant. We do show an Action Failed message on the console via [`CommandExecutionResult` component](https://github.com/elastic/kibana/blob/6faadda1eb78788114e1f530dcdd6718a252afa5/x-pack/plugins/security_solution/public/management/components/console/components/command_execution_result.tsx#L99). Keeping this extra logic will show a redundant Action Failed message on the history log below `<commandname> failed` message. Thus is made sense to remove it.

Reply:
Ok. so the response console handles it, but what happens in this case on the display in the Action History log? Will it just not show anything?

Maybe the message should be different? like "Unknown error!"?

Reply:
`Unknown error` is better